src/schema/rules/metadata/entities.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
This isn't anywhere close to something usable, but I'm not sure what we can do here.